src/main/java/String/GuessGame.java

```diff
@@ -0,0 +1,42 @@
+package String;
+
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class GuessGame {
+    public record GuessResult(boolean result, String hint) {
+    }
+
+    private static int hintCounter = 0;
+
+    public static GuessResult guessWordProcessing(String answer, String wordToGuess, StringBuilder hint) {
```

Comment:
```suggestion
    public static GuessResult guessWordProcessing(String answer, String wordToGuess) {
```
навіщо hint в аргументах? тим більше як StringBuilder